locales/de_DE/pcsx2_Iconized.po

```diff
@@ -540,87 +567,103 @@ msgstr ""
 "Abbrechen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:29
-#, fuzzy
 msgid ""
 "-3 - Reduces the EE's cyclerate to about 50%.  Big speedup, but *will* cause "
 "stuttering audio on many FMVs."
-msgstr "Stufe 3."
+msgstr ""
+"-3 - Reduziert die Zyklusrate des EE auf ungefähr 50%.  Großer FPS-Zuwachs, "
+"aber wird in FMVs Stottern verursachen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:34
-#, fuzzy
 msgid ""
 "-2 - Reduces the EE's cyclerate to about 60%.  Moderate speedup, but may "
 "cause stuttering audio on many FMVs."
-msgstr "Stufe 3."
+msgstr ""
+"-2 - Reduziert die Zyklusrate des EE auf ungefähr 60%.  Moderater FPS-"
+"Zuwachs, aber wird in vielen FMVs Stottern verursachen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:39
-#, fuzzy
 msgid ""
 "-1 - Reduces the EE's cyclerate to about 75%.  Mild speedup for most games "
 "with high compatibility."
-msgstr "Stufe 2."
+msgstr ""
+"-1 - Reduziert die Zyklusrate des EE auf ungefähr 75%.  Geringer FPS-Zuwachs "
+"für viele Spiele bei hoher Kompatibilität."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:45
-#, fuzzy
 msgid ""
 "0 - Default cyclerate (100%). This closely matches the actual speed of a "
 "real PS2 EmotionEngine."
-msgstr "Deaktiviert"
+msgstr ""
+"0 - Standart-Zyklusrate (100%). Dies entspricht sehr nah der originalen PS2 "
```

Comment:
nah? Vielleicht "in etwa" oder "ungefähr"? Oder "Es kommt [...] nah"? Aber klingt seltsam.